pkg/gather/clusterconfig/clusterconfig.go

```diff
@@ -427,6 +436,37 @@ func GatherClusterOperators(i *Gatherer) func() ([]record.Record, []error) {
 	}
 }
 
+func collectClusterOperatorResources(i *Gatherer, co configv1.ClusterOperator) *clusterOperatorResource {
+	var relObj configv1.ObjectReference
```

Comment:
What using *configv1.ObjectReference.
Later you could check if relObj == nil instead of creating empty struct